salt/modules/kubernetes.py

```diff
@@ -78,6 +82,25 @@ def __virtual__():
     return False, 'python kubernetes library not found'
 
 
+class TimeoutException(Exception):
+    pass
+
+
+if not platform.startswith("win"):
+    @contextmanager
+    def _time_limit(seconds):
+        def signal_handler(signum, frame):
+            raise TimeoutException
+        signal.signal(signal.SIGALRM, signal_handler)
+        signal.alarm(seconds)
+        try:
+            yield
+        finally:
+            signal.alarm(0)
+
+    _polling_time_limit = 30
```

Comment:
What do you think about converting this `_polling_time_limit` into a constant like `POLLING_TIME_LIMIT` ?

Reply:
Sure, why not!